simple/test.js

```diff
@@ -2934,106 +2932,135 @@ exports.test = function (global) {
-            params: { func: genericSatelliteGraph,
-              queryString: `
+        }
+      ];
+      let satelliteGraphTests = [
+        {
+          name: "aql-traversal-index-join",
+          params: {
+            func: genericSatelliteGraph,
+            queryString: `
                         FOR v, e, p IN 1..3 OUTBOUND CONCAT(@v, "/smart0:test0") GRAPH @g
                           FOR doc in @@c
                             FILTER doc.value1 == v.value1
                             RETURN doc
                       ` }
-          },
-          {
-            name: "aql-traversal-graph",
-            params: { func: genericSatelliteGraph,
-              queryString: `
+        },
+        {
+          name: "aql-traversal-graph",
+          params: {
+            func: genericSatelliteGraph,
+            queryString: `
                         FOR v, e, p IN 1..3 OUTBOUND CONCAT(@v, "/smart0:test0") GRAPH @g
                           return v`,
-              bindParamModifier: function (param, bindParam) {
-                delete bindParam["@c"];
-              }
+            bindParamModifier: function (param, bindParam) {
+              delete bindParam["@c"];
             }
-          },
-          {
-            name: "aql-index-traversal-graph",
-            params: { func: genericSatelliteGraph,
-              queryString: `
+          }
+        },
+        {
+          name: "aql-index-traversal-graph",
+          params: {
+            func: genericSatelliteGraph,
+            queryString: `
                         for doc in @@c
                           filter doc.value1 >= 0 and doc.value1 <= 10
                           let vkey = CONCAT(@v,"/smart", doc.value3, ":test", doc.value3)
                           for v, e, p in 1..4 outbound vkey graph @g
                             return {doc, p}
                         `
-            }
-          },
-          {
-            name: "aql-enum-collection-traversal-graph",
-            params: { func: genericSatelliteGraph,
-              queryString: `
+          }
+        },
+        {
+          name: "aql-enum-collection-traversal-graph",
+          params: {
+            func: genericSatelliteGraph,
+            queryString: `
                         for doc in @@c
                           let vkey = CONCAT(@v,"/smart", doc.value3, ":test", doc.value3)
                           for v, e, p in 1..4 outbound vkey graph @g
                             filter v.value1 <= doc.value1
                             return {doc, p}
                         `
-            }
           }
+        }
+      ];
 
+      function mdiTest (params) {
+        let myOptimizer = { rules: [] };
+        let bindParam = { "@col": params.collection };
+        if ("bindParamModifier" in params) {
+          params.bindParamModifier(params, bindParam);
+        }
+        db._query(
+          params.queryString,
+          bindParam,
+          { optimizer: myOptimizer }
```

Comment:
It looks like `optimizer` is always `{ rules: [] }`, so we can as well omit it entirely to make things less verbose.